dev/doc/README.md

```diff
@@ -6,28 +6,37 @@ See the first section of [`INSTALL`](../../INSTALL).  Developers are
 recommended to use a recent OCaml version, which they can get through
 opam or Nix, in particular.
 
-## Building `coqtop`
-The general workflow is to first setup a development environment with
-the correct `configure` settings, then hacking on Coq, make-ing, and testing.
+## Building `coqtop` / `coqc` binaries
 
+The recommended development workflow is to rely on Dune.
+See [`build-system.dune.md`](build-system.dune.md).
+In the example below, you may omit `-f Makefile.dune` by setting
+`COQ_USE_DUNE=1`.
 
-This document will use `$JOBS` to refer to the number of parallel jobs one
-is willing to have with `make`.
+```
+$ git clone https://github.com/coq/coq.git
+$ cd coq
+$ make -f Makefile.dune # to get an idea of the available targets
+$ make -f Makefile.dune check # an example of superfast target
+$ dune exec -- dev/shim/coqc-prelude test.v # to run coqc on a test file
+$ make -f Makefile.dune world # build all the standard library
+$ _build/install/default/bin/coqide # run CoqIDE, the whole stdlib is available
```

Comment:
Perhaps you could add some more spaces here so the comments are more clearly distinct from the commands?

Reply:
There is already doc for some of the ones below; maybe it could be reused.

- "an example of superfast target" => "build all OCaml files as fast as possible"
- " to run coqc on a test file" => "update coqc and the prelude and compile file test.v"
- "build all the standard library" => "build coq and the complete stdlib and setup it for use under _build/install/default"
- "run CoqIDE, the whole stdlib is available" why not use the shim here? This won't work so well once the user edits some files.